ci/build_container/build_recipes/zlib.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+VERSION=1.2.11
+
+wget -O zlib-$VERSION.tar.gz https://github.com/madler/zlib/archive/v$VERSION.tar.gz
+tar xf zlib-$VERSION.tar.gz
+cd zlib-$VERSION
+./configure --prefix=$THIRDPARTY_BUILD
+make V=1 install
```

Comment:
I believe the `V=1` is increasing the verbosity. Since the output is by default hidden I'd omit the `V=1`. It can always be manually added if someone needs to troubleshoot in the future.